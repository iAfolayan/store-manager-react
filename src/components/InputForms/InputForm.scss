label {
    color: var(--white);
}

.form-group {
    -ms-flex-align: center;
}