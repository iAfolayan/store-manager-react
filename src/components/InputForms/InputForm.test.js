import React from 'react';
import { shallow } from 'enzyme';
import InputForm from '.';

describe('<InputForm />', () => {
  test('should rendered Login form without crashing', () => {
    const wrapper = shallow(
      <InputForm
        label="text"
        text="text"
        type="text"
        name="text"
        value="text"
        placeholder="text"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
