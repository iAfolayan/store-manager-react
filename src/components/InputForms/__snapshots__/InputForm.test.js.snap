// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputForm /> should rendered Login form without crashing 1`] = `
<div
  className="form-group"
>
  <label
    className="text-sm-left"
    htmlFor="text"
  >
    text
  </label>
  <input
    className="col col-md-12 form-control"
    name="text"
    onChange={null}
    placeholder="text"
    text="text"
    type="text"
    value="text"
  />
</div>
`;
